kloppy/_providers/impect.py

```diff
@@ -0,0 +1,43 @@
+from kloppy.config import get_config
+from kloppy.domain.models.impect.event import ImpectEventFactory
+from kloppy.infra.serializers.event.impect import (
+    ImpectDeserializer,
+    ImpectInputs,
+)
+from kloppy.domain import EventDataset, Optional, List, EventFactory
+from kloppy.io import open_as_file, FileLike
+
+
+def load(
+    event_data: FileLike,
+    lineup_data: FileLike,
+    event_types: Optional[List[str]] = None,
+    coordinates: Optional[str] = None,
+    event_factory: Optional[EventFactory] = None,
+) -> EventDataset:
+    """
+    Load Impect event data into a [`EventDataset`][kloppy.domain.models.event.EventDataset]
+
+    Parameters:
+        event_data: filename of json containing the events
+        lineup_data: filename of json containing the lineup information
+        event_types:
+        coordinates:
+        event_factory:
```

Comment:
Args:
        event_data: JSON feed with the raw event data of a game.
        lineup_data: JSON feed with the corresponding lineup information of the game.
        event_types: A list of event types to load.
        coordinates: The coordinate system to use.
        event_factory: A custom event factory.

    Returns:
        The parsed event data.